frontend/src/features/projects/common/interfaces.ts

```diff
@@ -219,7 +232,8 @@ export interface IProjectProperty {
   building?: IApiProperty;
 }
 
-export const initialValues: any = {
+export const initialValues: IProject = {
```

Comment:
Now whenever we get a project from the api we start with this project and overwrite all valued fields using ... syntax. This ensures that all fields have valid formik defaults without having to process each field individually, which is verbose and error prone.